internal/docker/docker.go

```diff
@@ -22,13 +22,13 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/googleapis/librarian/internal/config"
```

Comment:
CI appears to be broken, please run `go run golang.org/x/tools/cmd/goimports@latest -w . `

Reply:
Done.